hack/init_data_plane_gke.sh

```diff
@@ -44,8 +44,8 @@ PROJECT_ID=${4:-$(gcloud config get-value project)}
 echo "PROJECT_ID ${PROJECT_ID} is used for this configuration"
 
 if [[ $MODE == "default" ]]; then
-  # Grant iam.serviceAccountAdmin permission of the Google service account cre-dataplane
```

Comment:
we might need to change `K8S_SERVICE_ACCOUNT` in this script as well. It is using `K8S_SERVICE_ACCOUNT=${3:-$DEFAULT_K8S_SERVICE_ACCOUNT}` and `DEFAULT_K8S_SERVICE_ACCOUNT="default-cre-dataplane"`

Reply:
Maybe in following PR